src/main/java/com/google/cloud/genomics/dataflow/readers/bam/Reader.java

```diff
@@ -97,12 +113,18 @@ void openFile() throws IOException {
     }
   }
 
+  static boolean isWrongSequence(SAMRecord record, Filter filter, 
+      String referenceName) {
+    return 
+        (filter == Filter.UNMAPPED_ONLY && !record.getReadUnmappedFlag()) ||
+        (filter == Filter.MAPPED_ONLY && record.getReadUnmappedFlag()) || 
+        (filter != Filter.UNMAPPED_ONLY && 
```

Comment:
No, I said I'll actually upload the changes later today:), just was going
through comments.

Thanks and sorry for the typos,
  Ilia (from mobile)
On Jul 10, 2015 10:45, "Paul Grosu" <notifications@github.com> wrote:

> In
> src/main/java/com/google/cloud/genomics/dataflow/readers/bam/Reader.java
> <https://github.com/googlegenomics/dataflow-java/pull/107#discussion_r34360764>
> :
>
> > @@ -97,12 +113,18 @@ void openFile() throws IOException {
> >      }
> >    }
> >
> > +  static boolean isWrongSequence(SAMRecord record, Filter filter,
> > +      String referenceName) {
> > +    return
> > +        (filter == Filter.UNMAPPED_ONLY && !record.getReadUnmappedFlag()) ||
> > +        (filter == Filter.MAPPED_ONLY && record.getReadUnmappedFlag()) ||
> > +        (filter != Filter.UNMAPPED_ONLY &&
>
> Sorry @iliat <https://github.com/iliat>, where is the added comment? I
> think my eyes are playing tricks on me this morning, or I just didn't have
> enough coffee.
>
> thx,
> ~p
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/googlegenomics/dataflow-java/pull/107/files#r34360764>
> .
>